core/basicsettings.ml

```diff
@@ -41,6 +41,16 @@ module Sessions = struct
               |> convert parse_bool
               |> CLI.(add (long "session-exceptions"))
               |> sync)
+
+  let expose_session_fail =
+    Settings.(flag "expose_session_fail" ~default:true
```

Comment:
I think this should be an option rather than a flag?

Reply:
A flag is basically a convenience for an option over values of type bool. So this instance looks right to me.

Reply:
Oh, I see. I think my hesitation was that it ended up exposed at the command line (i.e., we have the option for `--expose_session_fail`) rather than being config only.